build/goTranspiler.ts

```diff
@@ -1560,6 +1560,8 @@ func (this *${className}) Init(userConfig map[string]interface{}) {
             let content = go.content;
             content = this.regexAll (content, [
                 [/(\w+) := new ccxt\.Exchange\(([\S\s]+?)\)/gm, '$1 := ccxt.NewExchange().(*ccxt.Exchange); $1.InitParent($2, map[string]interface{}{}, $1)' ],
+                [/exchange interface\{\}, /g,'exchange *ccxt.Exchange, '], // in arguments
+                [/ interface\{\}(?= \= map\[string\]interface\{\} )/g, ' map[string]interface{}'], // fix incorrect variable type
```

Comment:
there were cases when nested interface was transpiled with such line  `const opts interface{}= map[string]interface{}`, so this regex fixes.